examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -13,8 +13,7 @@
    "source": [
```

Comment:
<p><code>And 'body/strides' is an integer -- therefore, the same value is used for all convolutional layers</code></p><p> </p><p><code>Explain more keys: at least all of the used in the next cell: output, </code> initial_block/inputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Too much information for this tutorial.